cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Extensions.kt

```diff
@@ -261,8 +261,13 @@ enum class FailureReason {
  */
 data class NodePath(
     val nodes: List<Node>,
+    /** See [HasAssumptions.assumptions]. */
```

Comment:
see above

Reply:
If there is no previous doc on the function, there is a way to show the doc of the parent function, so I will remove this. In cases where the overriding field/method adds Doc, the parent Doc is not shown, so here there may be a benefit in having the reference. On the other hand one can use navigation shortcuts to traverse the parent implementations/declarations until finding the Documentation.